src/core/objpool.c

```diff
@@ -9,7 +9,7 @@
 void objpool_init(struct objpool* objpool)
 {
     memset(objpool->pool, 0, objpool->objsize * objpool->num);
-    memset(objpool->bitmap, 0, BITMAP_SIZE(objpool->num));
+    memset(objpool->bitmap, 0, BITMAP_SIZE_IN_GRANULE(objpool->num));
```

Comment:
I think the original code had a bug. Since we are using memset should be:

```suggestion
    memset(objpool->bitmap, 0, BITMAP_SIZE_IN_BYTES(objpool->num));
```

Reply:
Fix 76536e9e